.gitignore

```diff
@@ -1,18 +1,12 @@
-*~
-.#*
-*#
-model.log
-*/input.csv
-__pycache__
-.python-version
-*sqlite3
+*sqlite3*
 .coverage
-htmlcov
-staticfiles
-releases
 .env
+.envrc
 .idea/
-node_modules/
 /assets/dist
 /assets/stats.html
 /coverage
+htmlcov
+node_modules/
+releases
```

Comment:
This'll confuse us if we extract the releases into their own app.

Reply:
Future us problems 🙂